### PR TITLE
Some minor code improvements

### PR DESCRIPTION
**Pull request by azmeuk**
as discussed in #3

### CODE REVIEW

File: src/zc/FileStorage/tests.py [hunk 1]
```diff
@@ -23,14 +23,15 @@
 import ZODB.tests.testblob
 import doctest
 import pickle
```
Comment (azmeuk) [inline src/zc/FileStorage/tests.py]: Maybe we should use zodbpickle instead?

Comment (mgedmin) [reply src/zc/FileStorage/tests.py]: I think that will be a necessity if zc.FileStorage ever wants to support Python 3, like the regular ZODB.FileStorage does.

File: src/zc/FileStorage/tests.py [hunk 2]
```diff
@@ -522,18 +529,28 @@ def snapshot_in_time():
...
+    if data[:2] == ".h":
+        return data
+
+    return ".h" + data.encode("hex")
 
 
 def unhexer(data):
-    return data and (data[:2] == ".h" and data[2:].decode("hex") or data)
+    if not data:
+        return None
```
Comment (mgedmin) [inline src/zc/FileStorage/tests.py]: Maybe `return data` would be safer?  Consider `unhexer("")`: old code would return `""`, your code would return `None`.

**Review by mgedmin — COMMENTED** [review]
Mostly LGTM except that one change to unhexer() about which I'm feeling less certain.

File: src/zc/FileStorage/tests.py (hunk 2 above)
Comment (azmeuk) [reply src/zc/FileStorage/tests.py]: Good catch !

**Comment by azmeuk** [comment]
Here it is :)

File: src/zc/FileStorage/__init__.py [hunk 3]
```diff
@@ -271,12 +271,8 @@ def _txn_find(self, tid, stop_at_pack):
 
 sys.path[:] = %(syspath)r
 
-if sys.version_info.major == 2:
-    import cPickle as pickle
-else:
-    import pickle
-
 import zc.FileStorage
+from zodbpickle import pickle
```
Comment (icemac) [inline src/zc/FileStorage/__init__.py:275]: Is the import from line 27 done here again or do I miss something?

Comment (azmeuk) [reply src/zc/FileStorage/__init__.py:275]: Yes. zodbpickle is used instead of pickle

Comment (mgedmin) [reply src/zc/FileStorage/__init__.py:275]: @icemac: you're looking at a large string literal here, called `pack_script_template`.

**Review by mgedmin — APPROVED** [review]
LGTM

**Pull request merged**